packages/cli/src/utils/load-blitz.ts

```diff
@@ -1,7 +1,38 @@
 import {forceRequire} from './module'
+import path from 'path'
+import globby from 'globby'
+import pkgDir from 'pkg-dir'
 
-export const BLITZ_MODULE_PATHS = ['@prisma/client']
+const projectRoot = pkgDir.sync() || process.cwd()
+
+export const BLITZ_MODULE_PATHS = [
+  ...globby.sync(
+    [
+      'app/**/{queries,mutations}/*.{js,ts,tsx}',
+      '**/utils/*.{js,ts,tsx}',
+      'jobs/**/*.{js,ts,tsx}',
+      'integrations/**/*.{js,ts,tsx}',
```

Comment:
Do we want to consider adding some kind of config file for blitz in the future so that the dev can add additional paths to the projects?

Reply:
yes, I thought about this. We already have `blitz.config.js`, so we'd add a field there for this.